server/src/main/java/org/opensearch/plugins/SearchPlugin.java

```diff
@@ -138,6 +141,14 @@ default Map<String, Highlighter> getHighlighters() {
         return emptyMap();
     }
 
+    /**
+     * Allows plugins to create concurrent search deciders
+     * @return Collection or ConcurrentSearchDeciders
+     */
+    default Collection<ConcurrentSearchDecider> createConcurrentSearchDeciders() {
```

Comment:
```suggestion
    default Collection<ConcurrentSearchDecider> getConcurrentSearchDeciders() {
```

More in line with all the other plugin methods

Reply:
I think `getConcurrentSearchDecidersList` should be renamed to `getConcurrentSearchDeciders` as you are suggesting. 

Here is how allocation deciders are registered from plugins. I think we should be consistent with that
https://github.com/opensearch-project/OpenSearch/blob/e8d3e695112c56a70b363ed5601f3b0382632bbf/server/src/main/java/org/opensearch/plugins/ClusterPlugin.java#L61